pkg/model/manifest.go

```diff
@@ -161,6 +161,8 @@ func (m Manifest) LocalPaths() []string {
 	switch di := m.deployTarget.(type) {
 	case DockerComposeTarget:
 		return di.LocalPaths()
+	case LocalTarget:
```

Comment:
should we panic() in the default case, to prevent this from getting missed in the future?